etc/config/cmakescript.amazon.properties

```diff
@@ -1,13 +1,19 @@
-compilers=cmake-3_26_1:cmake-3_27_9:cmake-3_28_0
+compilers=cmake-3_26_4:cmake-3_27_9:cmake-3_28_4:cmake-3_29_2:cmake-importstd
 isSemVer=true
 baseName=cmake
-defaultCompiler=cmake-3_28_0
+defaultCompiler=cmake-3_28_4
 
-compiler.cmake-3_26_1.exe=/opt/compiler-explorer/cmake-v3.26.1/bin/cmake
-compiler.cmake-3_26_1.semver=3.26.1
+compiler.cmake-3_26_4.exe=/opt/compiler-explorer/cmake-v3.26.4/bin/cmake
+compiler.cmake-3_26_4.semver=3.26.4
 
 compiler.cmake-3_27_9.exe=/opt/compiler-explorer/cmake-v3.27.9/bin/cmake
 compiler.cmake-3_27_9.semver=3.27.9
 
-compiler.cmake-3_28_0.exe=/opt/compiler-explorer/cmake-v3.28.0/bin/cmake
-compiler.cmake-3_28_0.semver=3.28.0
+compiler.cmake-3_28_4.exe=/opt/compiler-explorer/cmake-v3.28.4/bin/cmake
+compiler.cmake-3_28_4.semver=3.28.4
+
+compiler.cmake-3_29_2.exe=/opt/compiler-explorer/cmake-v3.29.2/bin/cmake
+compiler.cmake-3_29_2.semver=3.29.2
+
+compiler.cmake-importstd.exe=/opt/compiler-explorer/cmake-importstd/bin/cmake
+compiler.cmake-importstd.semver=3.29.20240416
```

Comment:
What is this version?

Reply:
This is a build of `master` prior to the 3.30 release. `master` uses the date in `YYYYMMDD` format as the patch number to ensure monotonicity of version numbers across history. I think a few days after this is better as it has some bug fixes.

Reply:
Does that install in the regular way? I'll give it a quick go (and than can unrevert if so)

Reply:
It doesn't install in the regular way:
```
2024-04-29 21:46:50,313 lib.ce_install  INFO     tools/cmake 3.29.2 is already installed, skipping
Installing tools/cmake 3.29.20240416
2024-04-29 21:46:50,514 lib.installation_context ERROR    Failed to fetch https://github.com/Kitware/CMake/releases/download/v3.29.20240416/cmake-3.29.20240416-Linux-x86_64.tar.gz: <Response [404]>
```

so... how does one install it? There are two parts to adding tools to CE: the installation and the config. The installation is handled in the `infra` PR (as linked below).

Reply:
No, they don't get tags and releases. You can get the commits that bump the version number (effectively a "tag") with `git log --first-parent --max-parents=1 --format=%H\ %cd origin/master`.

Reply:
Then we'll have to remove this. Or you'll need to make a PR to our `infra` repo that knows how to install these versions